aplicaciones/estudiantes/serializers.py

```
@@ -36,10 +42,17 @@ class CreateEstudianteSerializer(serializers.ModelSerializer):
         required=False,
         write_only=True
     )
+    unidad_id = serializers.PrimaryKeyRelatedField(
+        queryset=UnidadEducativa.objects.all(),
+        source='unidad',
+        allow_null=True,
+        required=False,
+        write_only=True
+    )
 
     class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)